### PR TITLE
Feature/cas 30

### PR DESCRIPTION
**Pull request by KrzysztofDomanczyk**

### CODE REVIEW

File: app/Database/Migrations/2022-03-09-113000_add_full_text_search_indexes.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        $createQuery = <<<CODE_SAMPLE
+            ALTER TABLE {$prefix}episodes
+            DROP INDEX IF EXISTS  episodes_search;
+        CODE_SAMPLE;
+
+        $this->db->query($createQuery);
+
+        $createQuery = <<<CODE_SAMPLE
+            ALTER TABLE {$prefix}podcasts
+            DROP INDEX IF EXISTS  podcasts_searcg;
```
Comment (KrzysztofDomanczyk) [inline app/Database/Migrations/2022-03-09-113000_add_full_text_search_indexes.php:52]: literówka

**Comment by yassinedoghri** [comment]
Hi @KrzysztofDomanczyk, thank you for the PR!

Unfortunately **this repository is read-only**, thus PRs cannot be accepted. Could you please submit a Merge Request [in the original repository](https://code.castopod.org/adaures/castopod)?

Note that **you'll need to create an account**. Let me know when you do so I can give you access to fork the project.

**Comment by KrzysztofDomanczyk** [comment]
@yassinedoghri I have account there. @KrzysztofDomanczyk, please give me fork permission.

**Comment by yassinedoghri** [comment]
> @yassinedoghri I have account there. @KrzysztofDomanczyk, please give me fork permission.

Ok, I just gave you permission to fork.

**Comment by KrzysztofDomanczyk** [comment]
I created the merge request: https://code.castopod.org/adaures/castopod/-/merge_requests/290

**Comment by yassinedoghri** [comment]
Thank you @KrzysztofDomanczyk! I'll take a look on monday, I'm currently unavailable.

**Pull request closed without merging**